src/repositories/courses-repository.js

```diff
@@ -3,14 +3,16 @@
 const errors = require('@feathersjs/errors');
 const { Op } = require('sequelize');
 
-module.exports = function createCoursesRepository(models) {
+module.exports = function createCoursesRepository(sequelize) {
   const {
     Course,
-    CourseTeacher,
     Teacher,
-    CourseSpecialty,
     Specialty,
-  } = models;
+    Group,
+    Student,
+    LabReport,
+    LabTask,
+  } = sequelize.models;
 
   async function list(queryParams) {

```

Comment:
5 ifs into a function is a code smell for violation of SRP

Reply:
You repeat this code 6 times in other files, please make sure you reuse the code

Reply:
the code is reused as much as it's possible, but each if has different number of included models and different models, so i can't reduce the complexity by splitting it in functions; and we need these changes on master as soon as possible to be able to work in frontend ://